<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BluetoothDebugScreen</class>
 <widget class="QDialog" name="BluetoothDebugScreen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>272</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(49, 49, 49);</string>
  </property>
  <widget class="QLabel" name="titleBluetoothDebug">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>30</y>
     <width>231</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Roboto</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:#1788c3
</string>
   </property>
   <property name="text">
    <string>Bluetooth Debug Screen</string>
   </property>
  </widget>
  <widget class="QPushButton" name="backBluetoothDebug">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>38</width>
     <height>30</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">border:none;
border-radius:3px;
background-color: rgb(70, 70, 73);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../rsc/resource.qrc">
     <normaloff>:/icons/previous.png</normaloff>:/icons/previous.png</iconset>
   </property>
  </widget>
  <widget class="QListWidget" name="bluetoothListWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>461</width>
     <height>192</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> QScrollBar:vertical {
     border: none;
     width: 5px;
 }
 QScrollBar::handle:vertical {
     background:#1788c3;
     min-height: 20px;
 }
 QScrollBar::up-arrow:vertical, QScrollBar::down-arrow:vertical {
     border: 2px #1788c3;
     width: 5px;
     height: 5px;
     background: #1788c3;
 }</string>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
  </widget>
  <widget class="QPushButton" name="incrementButton">
   <property name="geometry">
    <rect>
     <x>418</x>
     <y>30</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QPushButton" name="decrementButton">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>30</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QLabel" name="textTransferSize">
   <property name="geometry">
    <rect>
     <x>372</x>
     <y>35</y>
     <width>41</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Roboto</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:#1788c3
</string>
   </property>
   <property name="text">
    <string>20</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../rsc/resource.qrc"/>
 </resources>
 <connections/>
</ui>
